dig-bench.txt

```diff
@@ -0,0 +1,369 @@
+Reference
```

Comment:
We should remove this file before merging. Feel free to copy-paste to the PR or so if you want to record the results.